controller_manager:
  ros__parameters:
    update_rate: 50 # [Hz]

    joint_state_broadcaster:
      type: joint_state_broadcaster/JointStateBroadcaster

    joint_trajectory_controller:
      type: joint_trajectory_controller/JointTrajectoryController

joint_state_broadcaster:
  ros__parameters: {}

joint_trajectory_controller:
  ros__parameters:
    joints:
      - l_shoulder_joint
      - l_biceps_joint
      - l_elbow_joint
      - r_shoulder_joint
      - r_biceps_joint
      - r_elbow_joint
      - l_hip_joint
      - l_thigh_joint
      - l_knee_joint
      - l_ankle_joint
      - l_foot_joint
      - r_hip_joint
      - r_thigh_joint
      - r_knee_joint
      - r_ankle_joint
      - r_foot_joint

    command_interfaces:
      - position

    state_interfaces:
      - position
      - velocity

    allow_partial_joints_goal: false
